phonon/update.py

```diff
@@ -34,7 +35,7 @@ class Update(object):
     """
 
     def __init__(self, process, _id, database='test', collection='test',
-                 spec=None, doc=None, init_cache=False, block=True, hard_session=TTL,
+                 spec=None, doc={}, init_cache=False, block=True, hard_session=TTL,
```

Comment:
Every doc in this case will have a reference to the same `doc`. Python does funny things with caching for default params.

```python
>>> def foo(a, b=[]):
...   b.append(a)
...   return b
...
>>>
>>> foo(1)
[1]
>>> foo(2)
[1, 2]
>>>
```